docs/data-sources/integrations.md

```diff
@@ -0,0 +1,86 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "apono_integrations Data Source - terraform-provider-apono"
+subcategory: ""
+description: |-
+  Use this data source to get list integrations.
+---
+
+# apono_integrations (Data Source)
+
+Use this data source to get list integrations.
+
+## Example Usage
+
+```terraform
+data "apono_integrations" "k8s_integrations" {
+  type = "k8s-roles"
+}
+
+data "apono_integrations" "prod_integrations" {
+  connector_id = "prod-us-east-1"
+}
+
+data "apono_integrations" "prod_mysql_integrations" {
+  type         = "mysql"
+  connector_id = "production-us-east-1"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `connector_id` (String) Integration connector id to filter by
+- `type` (String) Integration type to filter by
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
+- `integrations` (Attributes List) List of integrations (see [below for nested schema](#nestedatt--integrations))
+
+<a id="nestedatt--integrations"></a>
+### Nested Schema for `integrations`
+
+Optional:
+
+- `aws_secret` (Attributes) (see [below for nested schema](#nestedatt--integrations--aws_secret))
+- `gcp_secret` (Attributes) (see [below for nested schema](#nestedatt--integrations--gcp_secret))
+- `kubernetes_secret` (Attributes) (see [below for nested schema](#nestedatt--integrations--kubernetes_secret))
+- `metadata` (Map of String) Integration metadata
+
+Read-Only:
+
+- `connector_id` (String) Apono connector identifier
+- `id` (String) Integration identifier
```

Comment:
Id can be confusing because integration has id..
maybe we want to get an integration id
let's change the name to something like data_set_id.

Reply:
In this case `id` is the ID of the integration, on line 40 the `id` is the "ID" of the resulting data source.
I can't remove/change it as it's mandatory for the tests